tests/browserstack/specs/valuesMainTest.js

```diff
@@ -0,0 +1,210 @@
+      await valueLink.click(); // Clicks on first value
+    }
+    await simpleClick('backToLinkTabHeader'); // Clicks on "Back"
+    await simpleClick('backToLinkTabHeader'); // Clicks on "Back"
+    await simpleClick('valuesFollowedPreviewShowMoreId'); // Clicks on "Explore all values"
+    await simpleClick('issueFollowButton'); // Follow value
+    await simpleClick('toggle-button');
+    await selectClick('li=Unfollow'); // Unfollow value
+    await search(clearSearchIconSelector, 'i.fas.fa-search', 'search_input', sqlInjectionTest); // tests search functionality
+    await simpleClick('backToLinkTabHeader'); // Clicks on "Back"
+
+    // //////////////////////
+    // Test "Public Figures to Follow" section
+    let publicFigureHeader = await $('h2=Public Figures to Follow');
+    await publicFigureHeader.scrollIntoView(); // Scrolls to "Public Figures to Follow"
+    await browser.pause(PAUSE_DURATION_MICROSECONDS);
+    await simpleClick('publicFiguresToFollowPreviewShowMoreId'); // Click "Explore more public figures"
+    await simpleClick('backToLinkTabHeader'); // Clicks on "Back"
+    publicFigureHeader = await $('h2=Public Figures to Follow');
+    await publicFigureHeader.scrollIntoView(); // Scrolls to "Public Figures to Follow"
+    const publicFigureRecommendations = await $(publicFigureCards);
+    if (await publicFigureRecommendations.waitForExist({ timeout: 5000 })) { // Check for recommendations
+      await browser.pause(PAUSE_DURATION_MICROSECONDS);
+      await selectClick(publicFigureFollowButton); // Follow first public figure
+      await selectClick(publicFigureDropDownButton);
+      await selectClick(publicFigureUnfollow); // Unfollow first public figure
+      await selectClick(publicFigureDropDownButton);
+      await selectClick(publicFigureIgnore); // Ignore first public figure
+      await selectClick(publicFigureDropDownButton);
+      await selectClick(publicFigureUnignore); // Unignore first public figure
+      await readMore(publicFigureSection); // Clicks more and show less
+      await selectClick(publicFigureLink); // Clicks first public figure's link
+      await browser.url('https://quality.wevote.us/values');
+    }
+
+    // //////////////////////
+    // Tests endorsements and twitter sign in
+    if (isDesktopScreenSize) {                 // Only for desktop
+      await simpleClick('twitterSignIn-splitIconButton'); // Clicks on "Find Public Opinions"
+      await browser.url('https://quality.wevote.us/values');
+      await simpleClick('undefined-splitIconButton'); // Clicks on "Add Endorsements"
+      await browser.url('https://quality.wevote.us/values');
+    }
+
+    // //////////////////////
+    // Tests organizations to follow
+    await simpleClick('organizationsToFollowPreviewShowMoreId'); // Clicks on "Explore more organizations"
+    await simpleClick('backToLinkTabHeader'); // Clicks on "Back"
+    publicFigureHeader = await $('h2=Public Figures to Follow');
```

Comment:
This is another example of where to use an id, since text like "Public Figures to Follow" may get modified without notice, and then break the test.